cmd/agola/cmd/runlist.go

```diff
@@ -60,11 +66,22 @@ func init() {
 	cmdRun.AddCommand(cmdRunList)
 }
 
-func printRuns(runs []*gwapitypes.RunResponse) {
+func printRuns(runs []*runDetails) {
 	for _, run := range runs {
-		fmt.Printf("%s: Phase: %s, Result: %s\n", run.ID, run.Phase, run.Result)
-		for _, task := range run.Tasks {
+		fmt.Printf("%s: Phase: %s, Result: %s\n", run.runResponse.ID, run.runResponse.Phase, run.runResponse.Result)
+		for i, task := range run.runTaskResponse {
```

Comment:
Currently the tasks are printed with a random order due to random map access when populating the `run.runTaskResponse` slice. We should order them by their level and then by name like done in the web ui. We could do this in this PR or just adding a TODO and do it in another PR.